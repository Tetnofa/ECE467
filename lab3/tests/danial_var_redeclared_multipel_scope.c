// int x;
// int y;
// int a;
// int a;
//int arr[2];
//bool x;
int z;
int z;
int main() { 
    // int adawldhk; // shouldn't be called probably. NOT THE GLOBAL SCOPE
    // if (1) {
    //     int y;
    // }
    int x;
    int x;
    {
        int x;
        int y;
        int x;
        {
            int x;
            int x;
            int x;
        }
    }
    {
        int a;
        int b;
        {
            int a;
            int b;
            int c;
            {
                int a; 
                int c; 
                int d;
            }
        }
    }

    //int y;
}
void fcn () {
    int y;
    int z;
    int z;
}
int z;
int y;
int f;
bool fa () {
    int y;
    int z;
    int g;
}