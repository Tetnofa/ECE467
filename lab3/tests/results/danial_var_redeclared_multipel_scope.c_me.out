sema: 8:0 : redefinition of z
sema: 15:4 : redefinition of x
sema: 19:8 : redefinition of x
sema: 22:12 : redefinition of x
sema: 23:12 : redefinition of x
sema: 46:4 : redefinition of z
sema: 48:0 : redefinition of z
